.github/workflows/golangci-lint.yaml

```
@@ -51,6 +51,6 @@ jobs:
     steps:
       - uses: actions/setup-go@v4
         with:
-          go-version: "1.21.4"
+         go-version: "1.21.5"
```

```suggestion
         go-version: "1.21.5"
```
I first had it `1.21` but it CI was failing saying ` Fixed in: path/filepath@go1.21.5` so updated go to `1.21.5`